### PR TITLE
Tracks station renames.

### PR DESCRIPTION
**Pull request by Iamgoofball**

### CODE REVIEW

**Comment by optimumtact** [comment]
Feature
freeze

**Comment by ChangelingRain** [comment]
This is not a feature.

File: code/game/objects/items/charter.dm [hunk 1]
```diff
@@ -86,7 +86,7 @@
 	name = "station charter for [station_name()]"
 	desc = "An official document entrusting the governance of \
 		[station_name()] and surrounding space to Captain [uname]."
-
+	feedback_add_details("station_renames","[station_name()]")
```
Comment (Jordie0608) [inline code/game/objects/items/charter.dm]: Use `feedback_set_details` for this.

**Comment by ExcessiveUseOfCobblestone** [comment]
is this for logging or what? I don't see why it's needed except to have meme names on Atlanta's website

**Comment by Cyberboss** [comment]
Exactly

**Comment by Iamgoofball** [comment]
Word cloud of station names

**Comment by Niknakflak** [comment]
Station Names: ------------ Ckey:
Buttstation12  -------------- iamgoofball
Ih8adminz13 --------------- Iamgoofball

Good chance though

**Comment by Iamgoofball** [comment]
^

**Comment by ExcessiveUseOfCobblestone** [comment]
wouldn't that go under logging instead of feedback

**Comment by duncathan** [comment]
@Jordie0608 like this?

**Comment by Cyberboss** [comment]
Yes

**Comment by Iamgoofball** [comment]
thx

**Comment by AnturK** [comment]
@nfreader

**Comment by nfreader** [comment]
Booya <3

**Pull request merged**